packages/esm-patient-registration-app/src/patient-registration/input/basic-input/select/select-input.test.tsx

```diff
@@ -29,4 +29,14 @@ describe.skip('select input', () => {
 
     await waitFor(() => expect(input.value).toEqual(expected));
   });
+  it('optional get added if the input is not required', () => {
+    const input = render(
+      <Formik initialValues={{ select: '' }} onSubmit={null}>
+        <Form>
+          <SelectInput label="Select" name="select" options={['A Option', 'B Option']} />
+        </Form>
+      </Formik>,
+    );
+    expect(screen.getByLabelText('Select (optional)')).toBeInTheDocument();
```

Comment:
Shall we change this the other way around using toBe(labelName) instead of toBeInTheDocument

Reply:
@anjula-sack updated the code accordingly. Check once.